src/templates/server_status.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
Templates should contain a license header too.

Reply:
This is currently not the case in renku-ui --> https://github.com/SwissDataScienceCenter/renku-ui/blob/master/public/index.html

Also, I do not see putting a license header on HTML pages as a standard practice. (Check sites like github, docker, gitlab, stack overflow, etc.)

Reply:
Since it's a Jinja2 template you can include it as a comment.

```
{#
...
#}
```

Reply:
is it necessary? required? @erbou ?